src/components/ProjectSettingsTabs/GithubCommitQueue/getFormSchema.ts

```diff
@@ -0,0 +1,350 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import { AliasRow } from "../AliasRow";
+import { overrideRadioBox, radioBoxOptions } from "../utils";
+import { FormState } from "./types";
+
+export const getFormSchema = (
+  useRepoSettings: boolean,
+  gitHubWebhooksEnabled: boolean,
+  formData: FormState,
+  repoData?: FormState
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    definitions: {
+      aliasArray: {
+        type: "array" as "array",
+        items: {
+          type: "object" as "object",
+          properties: {
+            id: {
+              type: "string" as "string",
+            },
+            alias: {
+              type: "string" as "string",
+            },
+            variant: {
+              type: "string" as "string",
+              default: "",
+            },
+            variantTags: {
+              type: "array" as "array",
+              title: "Variant Tags",
+              items: {
+                type: "string" as "string",
+                title: "Variant Tag",
+                default: "",
+              },
+            },
+            task: {
+              type: "string" as "string",
+              default: "",
+            },
+            taskTags: {
+              type: "array" as "array",
+              items: {
+                type: "string" as "string",
+                title: "Task Tag",
+                default: "",
+              },
+            },
+          },
+        },
+      },
+    },
+    type: "object" as "object",
+    properties: {
+      github: {
+        type: "object" as "object",
+        title: "GitHub",
+        properties: {
+          gitHubWebhooksEnabled: {
+            type: "null",
+            title: "GitHub Webhooks",
+            description: `GitHub webhooks ${
```

Comment:
I think that message should be tweaked slightly since Git Tag Versions and Github Checks also aren't available. 
Maybe the messaging could either give the full list or say "Github Features"? (The latter would probably only be reasonable if the content was disabled and not hidden, since the user may not know what it is they're missing out on.)

Possibly we could also add a disclaimer "Please reach out to #evergreen-users if you are still experiencing issues.